mod-record-specifications-validator/src/test/java/org/folio/rspec/validation/MarcSpecificationGuidedValidatorTest.java

```diff
@@ -76,18 +76,54 @@ void testInvalidIndicatorValidation() {
     var validationErrors = validator.validate(marc4jRecord, getSpecificationWithIndicators());
 
     assertThat(validationErrors)
-      .hasSize(8)
+      .hasSize(17)
       .extracting(ValidationError::getPath, ValidationError::getRuleCode)
       .containsExactlyInAnyOrder(
         tuple("035[0]^2", MarcRuleCode.INVALID_INDICATOR.getCode()),
+        tuple("035[0]^2", MarcRuleCode.UNDEFINED_INDICATOR.getCode()),
```

Comment:
We should probably not do UNDEFINED_INDICATOR validation if INVALID_INDICATOR already failed. What do you think?

Reply:
Yeah, it's make sense. It will be more complex in code but clear from user point of view. I'll change it.